source/renderer/app/stores/WalletsStore.ts

```diff
@@ -837,25 +837,29 @@ export default class WalletsStore extends Store {
         : null;
     const wallet = this.active;
     if (!wallet) throw new Error('Active wallet required before sending.');
-    // @ts-ignore ts-migrate(1320) FIXME: Type of 'await' operand must either be a valid pro... Remove this comment to see the full error message
-    await this.sendMoneyRequest.execute({
-      address: receiver,
-      amount: parseInt(amount, 10),
-      passphrase,
-      walletId: wallet.id,
-      isLegacy: wallet.isLegacy,
-      assets: formattedAssets,
-      hasAssetsRemainingAfterTransaction,
-    });
-    this.analytics.sendEvent(
-      EventCategories.WALLETS,
-      'Transaction made',
-      'Software wallet'
-    );
-    this.refreshWalletsData();
-    this.actions.dialogs.closeActiveDialog.trigger();
-    this.sendMoneyRequest.reset();
-    this.goToWalletRoute(wallet.id);
+    try {
```

Comment:
Wrapping this in a `try` catch block was necessary because otherwise it crashed Daedalus … im not sure how it worked with other types of errors though 🤔